CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Not released
 
+## 0.14.1
```

Comment:
Remove line 5, just leave it as "Not released".  We'll package a few changes together before we release.